src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -48,6 +54,12 @@ public T FishForMessage<T>(Predicate<T> isMessage, TimeSpan? max = null, string
             return FishForMessage(isMessage: isMessage, max: max, hint: hint, allMessages: null);
```

Comment:
Why isn't this sync over async?

Reply:
It is but not directly - I will change that to call `async` version of itself!